src/coreclr/pal/prebuilt/inc/fxver.h

```diff
@@ -1,7 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#ifndef HOST_UNIX
 #include <verrsrc.h>
+#endif
```

Comment:
The custom .rc parser on Unix seems honoring #include rule, but does not looks into the actual content of macros. Do we need a comment for it?

Reply:
If this ifdef is tricky, a comment would be useful.

Reply:
I am not sure I understand what you mean by "doesn't inspect the actual values of the macros". The parser first runs the resource file through the C preprocessor, so it should process all macros should be expanded to their values during that phase.

Reply:
I didn't look depth about how the parser works and how it works, but it seems that palrt.h doesn't bring any macro used. I'll update the comment.